server/plugin/command/create_event.go

```diff
@@ -0,0 +1,98 @@
+package command
+
+import (
+	"time"
+
+	"github.com/mattermost/mattermost-plugin-msoffice/server/remote"
+	"github.com/mattermost/mattermost-plugin-msoffice/server/utils"
+)
+
+func (c *Command) createEvent(parameters ...string) (string, error) {
+	// if len(parameters) != 2 {
+	// 	// return "jason", errors.New("Please specify an issue kassignee>`.")
+	// 	return "Run `/msoffice` for general help or `/msoffice viewcal` to get calendar id", nil
+	// }
+
+	// subject := parameters[0]
+	// body := parameters[1]
+	// calId := parameters[2]
+
+	// parse start time
+	start := &remote.DateTime{
+		TimeZone: "Pacific Standard Time",
+		DateTime: time.Now().Format(time.RFC3339),
+	}
+
+	// parse end time
+	end := &remote.DateTime{
+		TimeZone: "Pacific Standard Time",
+		DateTime: time.Now().Add(time.Hour).Format(time.RFC3339),
```

Comment:
Is the plan to allow the user to define times from the slash command? That could get a little ugly to type out versus using a time picker component. Maybe we use slack attachments with the feature to allow them to use a time picker?

@crspeller Is this something currently supported in the framework out of the box (time picker in slack attachment)?

Reply:
This is a POC and imagine the best implementation will be to have `/msoffice createevent` open a modal so we can use a timepicker/selection boxes/radio buttons.